### PR TITLE
Feat authentication

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Authentication page added

Have used color variables in some parts as they were declared in last branch and it's not merged. Will make changes once that is merged.

- Sign-up
screenshot

- Login
screenshot (1)

- Logout
screenshot (2)

### CODE REVIEW

File: log-in.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, 
+    initial-scale=1.0">
+
+    <!-- Font Awesome -->
+    <script src="https://kit.fontawesome.com/8fb4d48a29.js" crossorigin="anonymous"></script>
```
Comment (Hamdrive) [inline log-in.html:11]: This could also be imported into your main CSS file

**Review by Hamdrive — COMMENTED** [review]
Really love the authentication modals created, learnt alot about how I can improve my own project!
Will just suggest to take care of code formatting, else everything is well done and ready to merge!

File: log-in.html (hunk 1 above)
Comment (hinanshisuthar) [reply log-in.html:11]: Yes true. Will do that.

**Review by Tanay-J — APPROVED** [review]
Loved the design and color scheme. The code looks good as well, I think you can merge right away.

**Pull request merged**